extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -385,10 +385,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       .find(`#input-container-bcc`)
       .css('display', isThere.bcc ? '' : 'none');
     this.view.S.cached('bcc').css('display', isThere.bcc ? 'none' : '');
-    this.view.S.cached('input_addresses_container_outer')
-      .children(`:not([style="display: none;"])`)
-      .last()
-      .append(this.view.S.cached('container_cc_bcc_buttons')); // xss-reinsert
+    this.view.S.cached('input_addresses_container_outer').children(':visible').last().append(this.view.S.cached('container_cc_bcc_buttons')); // xss-reinsert
```

Comment:
The tests failed because of this selector `:not([style="display: none;"])`, which was not written correctly. The display value can be both `none` or `none;`. 

Due to Prettier removing the ; in `display: none;`, the tests failed.

Reply:
Great finding 👏

How did you detect it, using semaphore logs or locally?

Reply:
Using SemaphoreCI log.
Actually I had to disable `tap` output to see correct results.

This `tap` result and `xunit` result doesn't help much actually.

Reply:
https://flowcrypt.semaphoreci.com/jobs/e7c70e14-24c8-4d6f-abe4-05079d15824c

Reply:
> This tap result and xunit result doesn't help much actually. (Doesn't even show any logs if tests timeout)

Probably we'll be able to improve test results in https://github.com/FlowCrypt/flowcrypt-browser/issues/5057

Reply:
Yeah. I hope so